packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -1032,21 +1051,28 @@ export class ReadAlongComponent {
     //this.alignment = this.urlTransform(this.alignment)
     //this.text = this.urlTransform(this.text)
     //this.cssUrl = this.urlTransform(this.cssUrl)
+
     // TO maintain backwards compatibility language code
-    if (this.language.length < 3) {
-      if (this.language.match("fr") != null) {
-        this.language = "fra";
-      } else if (this.language.match("es") !== null) {
-        this.language = "spa";
+    if (prefs.language.length < 3) {
+      if (prefs.language.match("fr") != null) {
```

Comment:
This is weird code. Why not `prefs.language !== "fr"`? @deltork ? Is this case insensitive?

Reply:
This allows the app to support both ISO **639-3** and ISO **639-1** (`"fra"` and `"fr"`) for maximum compatibility.

Reply:
This makes it case insensitive
```suggestion
      if (prefs.language.match(/fr/i) != null) {
```